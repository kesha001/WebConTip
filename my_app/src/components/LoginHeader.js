import React, { Component } from "react";

class LoginHeader extends Component{
    render(){
        return(
            <div>
            <div><ul id="login_header" className="topnav">
              <h1>ConTip</h1>
            </ul></div>
            </div>
        )
    }
}

export default LoginHeader;
