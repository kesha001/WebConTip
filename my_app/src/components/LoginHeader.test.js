import React from 'react';
import { render } from '@testing-library/react';
import ReactDOM from 'react-dom';
import LoginHeader from './LoginHeader';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<LoginHeader />, div);
  ReactDOM.unmountComponentAtNode(div);
});

describe('LoginHeader', () => {

  const LoginHeaderContainer = (<LoginHeader/>)

  it('renders properly', () => {
    expect(LoginHeaderContainer).toMatchSnapshot()
  })

  const component = shallow(<LoginHeader/>)

  it('header to be visible', () => {
    expect(component.find('h1').text()).toEqual('ConTip')
  })

  });