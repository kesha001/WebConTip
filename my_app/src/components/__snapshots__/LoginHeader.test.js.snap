// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginHeader renders properly 1`] = `<LoginHeader />`;
